portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,33 +50,57 @@ private String convertToJsonUsingGson(List<String> messages) {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get comments in datastore, by most recent order at the top.
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    Query query = new Query("Comment").addSort(COMMENT_DATE, SortDirection.DESCENDING);
```

Comment:
I think you could also make "Comment" a class constant. I'll leave this as optional (and with "limit" below) since you're only using them once.

As a note, I wouldn't expect you to make "text" or "application/json" into class comments because those are not fields/names that are controlled by your code.